### PR TITLE
kepubify: init at 2.3.3

### PR DESCRIPTION
**Pull request by jacobdillon**
Also added myself to the list of maintainers.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

File: pkgs/applications/misc/kepubify/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  pname = "kepubify";
+  version = "2.3.3";
+  rev = "v${version}";
```
Comment (alexarice) [inline pkgs/applications/misc/kepubify/default.nix]: Is there a reason rev is here?

File: pkgs/applications/misc/kepubify/default.nix [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
```
Comment (alexarice) [inline pkgs/applications/misc/kepubify/default.nix]: You could just have lib as an argument instead and replace the `stdenv.lib` below by `lib`

File: pkgs/applications/misc/kepubify/default.nix [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    inherit rev;
+    sha256 = "1k8ips0dkbg607v81vjw3q86h3ls7mrpx62nhw6wsmix3cssms7y";
+  };
+
+  goDeps = ./deps.nix;
+
+  meta = with stdenv.lib; {
+    description = "Converts epubs to kepubs";
+    homepage = https://pgaskin.net/kepubify/;
+    license = stdenv.lib.licenses.mit;
```
Comment (alexarice) [inline pkgs/applications/misc/kepubify/default.nix]:
```suggestion
    license = licenses.mit;
```

File: pkgs/applications/misc/kepubify/default.nix [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    maintainers = with stdenv.lib.maintainers; [ jacobdillon ];
```
Comment (alexarice) [inline pkgs/applications/misc/kepubify/default.nix]:
```suggestion
    maintainers = with maintainers; [ jacobdillon ];
```

**Review by alexarice — COMMENTED** [review]
Could you rebase so that the maintainers commit comes before the adding the package commit. Further the commit to add yourself to the maintainers list should be titled:
`maintainers: add jacobdillon`

File: pkgs/applications/misc/kepubify/default.nix (hunk 1 above)
Comment (jacobdillon) [reply pkgs/applications/misc/kepubify/default.nix]: Nope, moved into the `src = fetchFromGithub {` block.

File: pkgs/applications/misc/kepubify/default.nix (hunk 2 above)
Comment (jacobdillon) [reply pkgs/applications/misc/kepubify/default.nix]: Good point, done

File: pkgs/applications/misc/kepubify/default.nix (hunk 3 above)
Comment (jacobdillon) [reply pkgs/applications/misc/kepubify/default.nix]: Done!

File: pkgs/applications/misc/kepubify/default.nix (hunk 4 above)
Comment (jacobdillon) [reply pkgs/applications/misc/kepubify/default.nix]: Done!

**Review by alexarice — APPROVED** [review]
builds with nix review and resulting executable runs

**Pull request closed without merging**